pkgs/development/libraries/libtinfo/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, dpkg }:
+
+stdenv.mkDerivation {
+  dontBuild = true;
+  nativeBuildInputs = [ dpkg ];
+  name = "libtinfo";
+  version = "5.5.9";
+  src = fetchurl {
+    url = "http://ftp.de.debian.org/debian/pool/main/n/ncurses/libtinfo5_5.9+20140913-1+deb8u3_amd64.deb";
+    sha256 = "36a0f120da15f82c0c729535ac48d95e126c5d7fc2c4aceb94bf27ced5a4cecc";
+  };
+  unpackPhase = "dpkg-deb -x $src .";
+  installPhase = ''
+    ls -al
+    mkdir -p $out/lib
+    cp lib/x86_64-linux-gnu/* $out/lib
+  '';
+
+  meta = with stdenv.lib; {
+    description = "low-level terminfo library for terminal handling";
```

Comment:
No homepage?
I think this is part of ncurses, right? Can we build this from the ncurses derivation?

Reply:
You're right. I can just use `${pkgs.ncurses5}/lib/libtinfo.so.5`